### PR TITLE
docs(graphql): `interface resolvers` section

### PR DESCRIPTION
**Pull request by alitnk**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Docs
- [ ] Other... Please describe:

---

Everything is mentioned here: https://github.com/nestjs/docs.nestjs.com/issues/2092

### CODE REVIEW

**Review by jmcdo29 — APPROVED** [review]
LGTM

**Comment by alitnk** [comment]
any chance for this PR to get merged?
@kamilmysliwiec

File: content/graphql/interfaces.md [hunk 1]
```diff
@@ -66,6 +66,30 @@ export abstract class Book {
 }
 ```
 
+##### Interface resolvers
```
Comment (kamilmysliwiec) [inline content/graphql/interfaces.md:69]:
```suggestion
#### Interface resolvers
```

File: content/graphql/interfaces.md [hunk 2]
```diff
@@ -66,6 +66,30 @@ export abstract class Book {
 }
 ```
 
+#### Interface resolvers
+
+So far, using interfaces, you could only share field definitions with your objects. If you also want to share the actual field resolvers implementation, you can create a dedicated interface resolver, as follows:
+To do so, create a resolver and point its type to the interface, then define your methods using the `@ResolveField` decorator.
```
Comment (kamilmysliwiec) [inline content/graphql/interfaces.md:72]:
```suggestion
```

File: content/graphql/interfaces.md [hunk 3]
```diff
@@ -66,6 +66,29 @@ export abstract class Book {
...
+@Resolver(type => Character) // Reminder: Character is an interface
+export class CharacterInterfaceResolver {
+  @ResolveField(() => [Character])
+  friends(
+    @Parent() character, // Resolved object that implements Character
+    @Info() { parentType }, // Type of the object that implements Character
+    @Args('search', { type: () => String }) searchTerm: string,
+  ) {
+    // Get character's friends
+    return friendsFromDB;
```
Comment (kamilmysliwiec) [inline content/graphql/interfaces.md:85]:
```suggestion
    // Get character's friends
    return [];
```

File: content/graphql/interfaces.md [hunk 4]
```diff
@@ -66,6 +66,29 @@ export abstract class Book {
...
+    @Info() { parentType }, // Type of the object that implements Character
+    @Args('search', { type: () => String }) searchTerm: string,
+  ) {
+    // Get character's friends
+    return [];
+  }
+}
+```
+
+Now the `friends` query is applied on all objects that implement the `Character` interface.
```
Comment (kamilmysliwiec) [inline content/graphql/interfaces.md:90]:
```suggestion
Now the `friends` query is auto-registered for all object types that implement the `Character` interface.
```

**Pull request merged**